Deepgram/Transcription/PrerecordedTranscriptionOptions.cs

```diff
@@ -1,141 +1,147 @@
+        ///  and X is the version number.
+        /// </summary>
+        [JsonProperty("diarize_version ")]
+        public string DiarizationVersion { get; set; } = null;
+
+        /// <summary>
+        /// Indicates whether to recognize alphanumeric strings. When set to true, whitespace will be removed
+        /// between characters identified as part of an alphanumeric string. 
+        /// </summary>
+        [JsonProperty("ner")]
+        public Nullable<bool> NamedEntityRecognition { get; set; } = null;
+
+        /// <summary>
+        /// Indicates whether to transcribe each audio channel independently.
+        /// </summary>
+        [JsonProperty("multichannel")]
+        public Nullable<bool> MultiChannel { get; set; } = null;
+
+        /// <summary>
+        /// Maximum number of transcript alternatives to return.
+        /// </summary>
+        [JsonProperty("alternatives")]
+        public Nullable<int> Alternatives { get; set; } = null;
+
+        /// <summary>
+        /// Indicates whether to convert numbers from written format (e.g., one) to numerical format (e.g., 1).
+        /// </summary>
+        [JsonProperty("numerals")]
+        public Nullable<bool> Numerals { get; set; } = null;
+
+        /// <summary>
+        /// Terms or phrases to search for in the submitted audio.
+        /// </summary>
+        [JsonProperty("search")]
+        public string[] SearchTerms { get; set; }
+
+        /// <summary>
+        /// Terms or phrases to search for in the submitted audio and replace.
+        /// </summary>
+        /// <remarks>Phrases should be submitted in the format: searchfor:replacewith</remarks>
+        [JsonProperty("replace")]
+        public string[] Replace { get; set; }
+
+        /// <summary>
+        /// Callback URL to provide if you would like your submitted audio to be processed asynchronously.
+        /// When passed, Deepgram will immediately respond with a request_id. 
+        /// </summary>
+        [JsonProperty("callback")]
+        public string Callback { get; set; } = null;
+
+        /// <summary>
+        /// Keywords to which the model should pay particular attention to boosting or suppressing to help
+        /// it understand context.
+        /// </summary>
+        [JsonProperty("keywords")]
+        public string[] Keywords { get; set; }
+
+        /// <summary>
+        /// Indicates whether Deepgram will segment speech into meaningful semantic units, which allows
+        /// the model to interact more naturally and effectively with speakers' spontaneous speech patterns.
+        /// </summary>
+        [JsonProperty("utterances")]
+        public Nullable<bool> Utterances { get; set; } = null;
+        
+        /// <summary>
+        /// Indicates whether to detect the language of the provided audio.
+        /// </summary>
+        [JsonProperty("detect-language")]
```

Comment:
do you want this to be a dash or an underscore?

Reply:
Good catch